packages/office-ui-fabric-react/src/components/Button/DefaultButton/DefaultButton.tsx

```diff
@@ -1,11 +1,17 @@
+import { css } from '../../../Utilities';
 import { BaseButton } from '../BaseButton';
-import './DefaultButton.scss';
-import '../ButtonCore/ButtonCore.scss';
+import styles from './DefaultButton.scss';
 
 export class DefaultButton extends BaseButton {
-  protected _variantClassName = 'ms-Button--default';
+  protected classNames = {
```

Comment:
Why classNames? it seems you are passing styles in here for various elements, right?